devlib/module/cgroups.py

```diff
@@ -287,6 +287,11 @@ def get_tasks(self):
         logging.debug('Tasks: %s', task_ids)
         return map(int, task_ids)
 
+    # Used to insert fake cgroup attach events to know existing cgroup assignments
+    def trace_cgroup_tasks(self):
```

Comment:
We are already into the CGroup class, thous I would suggest to rename this to be just `CGroup::trace_tasks`